### ISSUE TITLE
Cannot import Facebook as global

### ISSUE THREAD

**Issue by kayue** [systemjs/systemjs#963]
I am getting the following error when `System.import('facebook');`

```
https://connect.facebook.net/en_US/all.js did not call System.register or AMD define
```

I have already defined it as global format in config:

```
meta: {
    "facebook": {
        exports: 'FB',
        format: 'global'
    }
},
map: {
    "facebook": "https://connect.facebook.net/en_US/all",
}
```

I am using `system-csp-production.js`. I used to have this kind of syntax working in RequireJS. May I know is this possible in System.js?

**Comment by guybedford** [comment]
Thanks, this is a bug. Fixed in https://github.com/systemjs/systemjs/commit/2863a7987cacb5151fd08842e6a2fd9890040386.

**Comment by guybedford** [comment]
Released in 0.19.7.
